src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
```

Comment:
lets add private constructor

Reply:
I added constructor without fields, but i don't no why in this class ?